src/utils/constants/index.js

```diff
@@ -80,6 +80,8 @@ const CHANNEL_LIST_GET_INITIAL_MESSAGES_LIMIT = 20;
 const SETTINGS_FAQ_URL = 'https://home.helio.social/faq';
 const SETTINGS_PRIVACY_URL = 'https://home.helio.social/privacy';
 
+const CONTACT_SCREEN_MAX_SELECTED_USERS = 99;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The constant `CONTACT_SCREEN_MAX_SELECTED_USERS` is defined here, but it would be better to add a comment explaining what this constant is used for. This will improve the readability and maintainability of the code.
```diff
+ // Maximum number of users that can be selected on the contact screen
  const CONTACT_SCREEN_MAX_SELECTED_USERS = 99;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->